src/ui/components/Collections.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import Select from 'react-select';
+import axios from 'axios';
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow
+} from '@material-ui/core';
+import ResourceCell from './ResourceCell';
+
+const Collections = () => {
+  const [data, setData] = useState({ collectionName: '', headers: [], data: [] });
+  const collections = [
+    { value: 'servers', label: 'Servers' },
+    { value: 'endpoints', label: 'Endpoints' },
+    { value: 'subscriptions', label: 'Subscriptions' },
+    { value: 'plandefinitions', label: 'PlanDefinitions' },
+    { value: 'logs', label: 'Logs' }
+  ];
+
+  const onSelect = selectedOption => {
+    axios.get(`http://localhost:3000/${selectedOption.value}`).then(response => {
```

Comment:
Consider using react-query `useQuery` here. It won't really change anything now but it will allow us to easily update the display when the data changes.

Reply:
Updated to use this!